microshift_release_notes/microshift-4-14-release-notes.adoc

```diff
@@ -490,4 +458,11 @@ Issued: 2024-03-20
 
 {product-title} release 4.14.17 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2024:1262[RHBA-2024:1262] advisory. The images that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2024:1260[RHBA-2024:1260] advisory.
 
-For the `TopoLVM image`, see link:https://catalog.redhat.com/software/containers/lvms4/topolvm-rhel9/63972de3adcb55263891b983?container-tabs=dockerfile[lvms4/topolvm-rhel9].
\ No newline at end of file
+For the `TopoLVM image`, see link:https://catalog.redhat.com/software/containers/lvms4/topolvm-rhel9/63972de3adcb55263891b983?container-tabs=dockerfile[lvms4/topolvm-rhel9].
+
+[id="microshift-4-14-18-dp"]
+=== RHBA-2024:1460 - {microshift-short} 4.14.18 bug fix update advisory
+
+Issued: 2024-03-27
+
+{product-title} release 4.14.18 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2024:1460[RHBA-2024:1460] advisory. The images that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:1458[RHSA-2024:1458] advisory.
```

Comment:
FYI links going to 404. Like the comment above, may be expected given nature of RNs, but just in case wanted to mention.

Reply:
Yeah, that's as it should be. There's no page until after the release. If you want to go an extra mile to check the links, you can look at the Errata (but that's a big extra mile).

Reply:
> Yeah, that's as it should be. There's no page until after the release. If you want to go an extra mile to check the links, you can look at the Errata (but that's a big extra mile).

Thanks! I got my steps in yesterday so I'm good... :)